### ISSUE TITLE
Error printText image: [TypeError: null is not an object (evaluating 'RNPhotoManipulator.printText')]

### ISSUE THREAD

**Issue by saim683** [guhungry/react-native-photo-manipulator#855]
I have facing an issue of Error printText image: [TypeError: null is not an object (evaluating 'RNPhotoManipulator.printText')].
when i console RNPhotoManipulator i got the functions in the console.
{"RNPhotoManipulator": {"batch": [Function batch], "crop": [Function crop], "flipImage": [Function flipImage], "optimize": [Function optimize], "overlayImage": [Function overlayImage], "printText": [Function printText], "rotateImage": [Function rotateImage]}}
But when I pass the URI ,text and mimeType. I got the error mentioned above.
here is my code 

const handleManipulation = async imageUri => {
    try {
      const mimeType = 'image/jpeg';
      const image =
        'https://unsplash.com/photos/qw6qQQyYQpo/download?force=true';
      const texts = [
        {
          position: {x: 50, y: 30},
          text: 'Text 1',
          textSize: 30,
          color: '#000000',
        },
        {
          position: {x: 50, y: 30},
          text: 'Text 1',
          textSize: 30,
          color: '#FFFFFF',
          thickness: 3,
        },
      ];

      await RNPhotoManipulator.printText(image, texts, mimeType).then(path => {
        console.log(`Result image path: ${path}`);
      });
    } catch (error) {
      console.error('Error cropping image:', error);
    }
  };

**Comment by guhungry** [comment]
Hi @saim683, I would like to ask you a few questions.

1. What is the version of this library you are using?
2. Are you using Expo?
3. What is your React Native version?

**Comment by saim683** [comment]
Hi Guhungry/React-Native-Photo-Manipulator.
The answers to the questions are as follows.
1. I am using the 1.5.2 version of the library.
2. No, I am using react-native cli.
3. I am using the 0.64.4 version of the react native cli

Thank you for asking

**Comment by guhungry** [comment]
Hello @saim683 again,

Which platforms are you having problem? Android, ios or both

**Comment by guhungry** [comment]
Hi @saim683, It should be fixed in [v1.5.3](https://github.com/guhungry/react-native-photo-manipulator/releases/tag/v1.5.3) 🎉 🎉 🎉 🎉 🎉 

If you still have a problem feel free to repoen this issue

**Comment by saim683** [comment]
Hi Guhungry/React-Native-Photo-Manipulator
Hope you are doing well.
After installing the latest version my app didn't run. It returns the error
of
Execution failed for task
':react-native-photo-manipulator:compileDebugKotlin
And my kotlin version is 1.5.10.
I updated my kotlin_version to '1.5.31'. But I'm still getting the same
error.

**Comment by guhungry** [comment]
Hi @saim683
Can you update kotlin version to 1.9.24? You also need to update gradle wrapper to 6.8.3 too.

**Comment by saim683** [comment]
Hi Guhungry/React-Native-Photo-Manipulator

So there is an issue that when i update my kotlin version and gradle
wrapper my app doesn't run. Many packages are not supported to the
latest version of kotlin and gradle wrapper. Is there any other possible
way to fix this?

Thank you for guidance

**Comment by guhungry** [comment]
Hi @saim683,

It should be fixed in [v1.5.4](https://github.com/guhungry/react-native-photo-manipulator/releases/tag/v1.5.4) 🎉 🎉 🎉 🎉 🎉 


> Hi Guhungry/React-Native-Photo-Manipulator Hope you are doing well. After installing the latest version my app didn't run. It returns the error of Execution failed for task ':react-native-photo-manipulator:compileDebugKotlin And my kotlin version is 1.5.10. I updated my kotlin_version to '1.5.31'. But I'm still getting the same error.

But you might need to config kotlin version with `kotlinVersion` too, so in your case it should be
```gradle
buildscript {
    ext {
        ...
        kotlin_version = "1.5.10"
        kotlinVersion = "1.5.10"
    }
    ...
```

**Comment by saim683** [comment]
Thanks, it worked.
